kopi/assetlist.json

```diff
@@ -0,0 +1,497 @@
+      "denom_units": [
+        {
+          "denom": "ibc/DE63D8AC34B752FB7D4CAA7594145EDE1C9FC256AC6D4043D0F12310EB8FC255",
+          "exponent": 0
+        },
+        {
+          "denom": "INJ",
+          "exponent": 18
+        }
+      ],
+      "base": "ibc/DE63D8AC34B752FB7D4CAA7594145EDE1C9FC256AC6D4043D0F12310EB8FC255",
+      "name": "Injective",
+      "display": "INJ",
+      "symbol": "INJ",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/inj.svg"
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/inj.svg"
+        }
+      ],
+      "keywords": [
+        "dex",
+        "mm"
+      ],
+      "type_asset": "ics20",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "injective",
+            "base_denom": "inj",
+            "channel_id": "channel-376"
+          },
+          "chain": {
+            "channel_id": "channel-1",
+            "path": "transfer/channel-1/inj"
+          }
+        }
+      ]
+    },
+    {
+      "description": "Luna from Terra2 Chain",
+      "denom_units": [
+        {
+          "denom": "ibc/DA59C009A0B3B95E0549E6BF7B075C8239285989FF457A8EDDBB56F10B2A6986",
+          "exponent": 0
+        },
+        {
+          "denom": "Luna",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/DA59C009A0B3B95E0549E6BF7B075C8239285989FF457A8EDDBB56F10B2A6986",
+      "name": "Luna",
+      "display": "luna",
+      "symbol": "LUNA",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/luna.svg"
+      },
+      "images": [
+        {
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terra2/images/luna.svg"
+        }
+      ],
+      "keywords": [
+        "dex",
+        "mm"
+      ],
+      "type_asset": "ics20",
+      "traces": [
+        {
+          "type": "ibc",
+          "counterparty": {
+            "chain_name": "terra2",
+            "base_denom": "uluna",
+            "channel_id": "channel-586"
+          },
+          "chain": {
+            "channel_id": "channel-5",
+            "path": "transfer/channel-5/uluna"
+          }
+        }
+      ]
+    },
+    {
+      "description": "Osmosis from Osmosis Chain",
+      "denom_units": [
+        {
+          "denom": "ibc/646315E3B0461F5FA4C5C8968A88FC45D4D5D04A45B98F1B8294DD82F386DD85",
+          "exponent": 0
+        },
+        {
+          "denom": "Luna",
+          "exponent": 6
+        }
+      ],
+      "base": "ibc/646315E3B0461F5FA4C5C8968A88FC45D4D5D04A45B98F1B8294DD82F386DD85",
+      "name": "OSMO",
+      "display": "Osmosis",
+      "symbol": "OSMO",
+      "logo_URIs": {
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/Osmosis/images/osmo.svg"
```

Comment:
capital 'O' in Osmosis should be lowercase

Reply:
Updated the file for osmosis and Luna

Reply:
In the svg path, specifically. The referenced directory name /Osmosis/ does not exist. it should be changed to /osmosis/ (with a lowercase 'o')

Reply:
Image path has been updated